package.json

```diff
@@ -27,5 +26,8 @@
     "mocha": "^7.1.2"
   },
   "author": "Jörg Tillmann <joerg@peaksandpies.com>",
-  "license": "MIT"
+  "license": "MIT",
+  "engines": {
+    "node": ">=12.18.2"
```

Comment:
Is there a reason this is needed?

Reply:
Other then making sure ES6 works, no.

Reply:
Folks might be using this library with pre-ES6 environments. Should be a separate change and separate discussion?

Reply:
12 seems like a big jump. While [12 is the current active and 10 is in maintenance](https://nodejs.org/en/about/releases/), I see no reason in this PR to require anything greater than node 10. [Node 10 supports all ES2017 features](https://node.green/#ES2017) (which there isn't a whole lot of use of here).

If keeping support for pre-es6 (ES2015) is important, then lowering the required version further would be required, I imagine.

(Selfishly: libraries I maintain need to continue to support node 10 for now, and this change would cause headaches while trying to move off `request`)